zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/RoleHandler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import io.camunda.exporter.exceptions.PersistenceException;
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.usermanagement.RoleEntity;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.value.RoleRecordValue;
+import java.util.List;
+
+public class RoleHandler implements ExportHandler<RoleEntity, RoleRecordValue> {
+  private final String indexName;
+
+  public RoleHandler(final String indexName) {
+    this.indexName = indexName;
+  }
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.USER;
+  }
+
+  @Override
+  public Class<RoleEntity> getEntityType() {
+    return RoleEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(final Record<RoleRecordValue> record) {
+    return getHandledValueType().equals(record.getValueType());
+  }
+
+  @Override
+  public List<String> generateIds(final Record<RoleRecordValue> record) {
+    return List.of(String.valueOf(record.getKey()));
+  }
+
+  @Override
+  public RoleEntity createNewEntity(final String id) {
+    return new RoleEntity().setId(id);
+  }
+
+  @Override
+  public void updateEntity(final Record<RoleRecordValue> record, final RoleEntity entity) {
+    final RoleRecordValue value = record.getValue();
+    entity
+        .setRoleKey(value.getRoleKey())
+        .setName(value.getName())
+        .setEntityKey(value.getEntityKey());
```

Comment:
Need to expand on the setEntityKey, add/remove to the list of linked entities. Not a straight setter, some manipulation will be involved